### DISCUSSION TITLE
How to scale the UI under Linux (Flatpak)

### DISCUSSION THREAD

**Discussion by HubKing** [tamlok/vnote#2495] [Q&A] [answered]
Wayland Plasma 6, no system scaling for X11. The UI looks very small. In the settings, I could enlarge the toolbar, but I could not find a way to scale the rest of UI. How to enlarge the entire UI?

![image](https://github.com/vnotex/vnote/assets/10900489/7ebf6d56-466c-48c2-a5ad-8ae5573d454b)

**Comment by tamlok** [comment, marked as answer]
What is your system version and monitor config? Qt should scaling the APP automatically.
